ppr-ui/src/components/tables/mhr/HomeSectionsTable.vue

```diff
@@ -173,12 +231,26 @@ export default defineComponent({
     }
     const isActiveIndex = (index: number): boolean => { return index === localState.activeIndex }
 
+    const showCorrectUndoOptions = (item: HomeSectionIF): boolean => {
+      return [ActionTypes.REMOVED, ActionTypes.CORRECTED].includes(item?.action)
+    }
```

Comment:
No change required here as it may not be applicable in this context but want to share some thoughts. 

A function expression is not inherently reactive, only evaluated when invoked (which it may repeated in the table, i dunno). To me this seems like it makes the most sense as a computed value because item.action could be dynamic, and subsequently a dynamic return value. 

Just some food for thought, thats how i generally differentiate between a computed and function expression. Do i want to react to some derived value (ie item.action) or call something once via some event etc.

Like i said, zero action required here, i'm sure it works, just sharing my thoughts on this!
I'm a little biased towards computed values just cuz it leverages so much of the frameworks reactivity inherently.

Reply:
Good point, let me take another look. I referenced `HomeOwnersTable` as it has the same function with the same purpose. Maybe in HomeSections it's a bit different...

Reply:
For sure, in the tables it's probably being invoked on an update somehow, i'm not really sure. 

Just sharing some high level thoughts, when considering what to use between function expression vs computed. 
No action required at all!

Reply:
I think it should be ok. The computed `homeSections` contains the `item: HomeSectionIF`. If it gets updated, then the row will be re-rendered and the `showCorrectUndoOptions()` would be called again. I'm happy to make an update.